distribution/changelog.txt

```diff
@@ -33,10 +33,12 @@
 - Fix: [#7544] Starting a headless server with no arguments causes the game to freeze.
 - Improved: [#2989] Multiplayer window now changes title when tab changes.
 - Improved: [#5339] Change eyedropper icon to actual eyedropper and change cursor to crosshair.
+- Improved: [#5832] Resize tile inspector automatically when selecting a tile element.
 - Improved: [#7302] Raising land near the map edge makes the affected area smaller instead of showing an 'off edge map' error.
 - Improved: [#7435] Object indexing now supports multi-threading.
 - Improved: [#7510] Add horizontal clipping to cut-away view options.
 - Improved: [#7531] "Save track design" dropdown now stays open.
+- Improved: [#7548] Add Ctrl + Click functionality to select tile elements with the tile inspector.
```

Comment:
Could you clarify here how this new function differs from the old one?

Reply:
I find it difficult to summerize it in few words. The current way of selecting elements is first clicking on the tile, then on the element in the list. With this you can Ctrl+click on the element in the viewport directly, and it will select the tile that the element is on and select the element in the list.

Reply:
Ah, that helps. Perhaps something like "Ctrl-clicking an element now directly selects it and its tile in the tile inspector."? Although that suggests you don't need to have the tile inspector open, perhaps.

On a side note, I think we've substituted Ctrl shortcuts with Cmd ones on macOS. Is that taken into account here?

Reply:
Thanks, that's better already. "Ctrl-Clicking with the tile inspector open now now directly selects the element and its tile." Perhaps using "Ctrl/Cmd + Clicking" instead it clearer for MacOS users? Or is it normal to refer to the Cmd key with Ctrl?

The code cheks for `PLACE_OBJECT_MODIFIER_COPY_Z` rather than for Ctrl input directly.

Reply:
> The code cheks for `PLACE_OBJECT_MODIFIER_COPY_Z` rather than for Ctrl input directly.

Ah, I don't think that's been adjusted for macOS. Should be fine to just refer to Ctrl, then. So, "Ctrl-clicking with the tile inspector open now directly selects an element and its tile."?

Reply:
Thanks! I've amended the commit.